controllers/fenceagentsremediation_controller_test.go

```diff
@@ -132,7 +114,7 @@ var _ = Describe("FAR Controller", func() {
 
 		When("creating valid FAR CR", func() {
 			BeforeEach(func() {
-				node = getNode(node01)
+				node = utils.GetNode("", node01)
```

Comment:
IIUC node is always initialized to the same value.
If this is indeed the case I suggest:
- initialize node value on var declaration
- move the node creation to a share BeforeEach block

Reply:
> IIUC node is always initialized to the same value.

Correct, but if it is initialized once then the second test would fail as the instance might be deleted or already created while the second test tries to create it (an error due race condition). To overcome it we initialize an independent instance of the node (same spec and name, but different **resource version**) in a different BeforeEach block.